tests/migration_test.go

```diff
@@ -2573,6 +2573,96 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				Entry("[sig-compute][test_id:3241]cancel a migration right after posting it", false),
 				Entry("[sig-compute][test_id:3246]cancel a migration with virtctl", true),
 			)
+
+			Context("[Serial]when target pod cannot be scheduled and is suck in Pending phase", func() {
```

Comment:
I think the commit could be more descriptive. Something like:
`Test: Migration is possible to be aborted when the target pod cannot be scheduled.`

Reply:
Done, thanks